primo/data_parser/well_data.py

```diff
@@ -37,6 +37,9 @@
 OWNER_WELL_COLUMN_NAME = "Owner Well-Count"
 
 
+# disabling too many lines in module warning
+# pylint: disable = C0302
```

Comment:
does too-many-lines as string not work? That is easier to interpret than having to check what C0302 means (although your comment above helps).

Reply:
You can also disable multiple pylint issues in one line as follows:
`#pylint: disable=too-many-lines, too-many-public-methods`